app/main.py

```diff
@@ -1,25 +1,80 @@
+from __future__ import annotations
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Visitor, name: str) -> None:
+        self.name = "_" + name
+
+    def __get__(
+            self,
+            instance: SlideLimitationValidator,
+            owner: Visitor
+    ) -> Visitor:
```

Comment:
Please, use more generic types in the type annotation to use `IntegerRange` class for classes other than indicated here as well (e.g. use `Any`)